content/creator/getting-started/index.mdx

```diff
@@ -0,0 +1,263 @@
+---
+date: 2019-01-01
+title: 'Getting Started as an egghead Content Creator'
+description: 'This article will help you orient yourself as a new egghead instructor. If you were recently invited to teach on egghead, or are curious about how content is produced on egghead, you are ihn the right place!'
+categories: ['creator', 'getting-started']
+published: false
+---
```

Comment:
there is no `slug` frontmatter as they utilize the file structure to determine the slugs. If a `slug` is available, it will override the filesystem behavior.